includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -2014,10 +2009,9 @@ public static function insert_setting_after( &$settings, $insert_after_setting_i
 	 * @param array $settings The list of settings
 	 */
 	public static function add_guest_checkout_setting_note( $settings ) {
-		$is_wc_pre_3_4_0 = wcs_is_woocommerce_pre( '3.4.0' );
 		$current_filter  = current_filter();
```

Comment:
Because the `$is_wc_pre_3_4_0 = wcs_is_woocommerce_pre( '3.4.0' );` was removed here, this `$current_filter line needs to be updated to remove the aligning space. 

```suggestion
		$current_filter = current_filter();
```